ledger/metrics/src/main/scala/com/daml/metrics/Timed.scala

```diff
@@ -32,6 +32,15 @@ object Timed {
     result
   }
 
+  def countedFuture[T](counter: Counter, future: => Future[T]): Future[T] = {
```

Comment:
Looks sensible. I'm not sure if "counted" is the right word; perhaps make it clear it's in progress?

That said, I can't think of a better option.

Reply:
Maybe `trackedInProgressFuture`?

Reply:
Perhaps `trackedFuture` is OK. Just to use a word that isn't overloaded.

Reply:
If naming is not sufficient, it's probably worth adding a comment on how this counter is different from the one that `Timer`s already keep. That would have been my only doubt, otherwise. LGTM. :slightly_smiling_face:

Reply:
I've renamed them to `trackedFuture` and `timedAndTrackedFuture`.